src/supertux/shrinkfade.cpp

```diff
@@ -41,7 +42,7 @@ void
 ShrinkFade::draw(DrawingContext& context)
 {
   float progress = m_accum_time / m_fade_time;
-  float diameter = 2 * m_initial_size * (1.0f - progress);
+  float diameter = 2 * m_initial_size * (m_direction == FADEOUT ? (1.0f - progress) : progress);
   context.color().draw_inverse_ellipse(m_dest, Vector(1.1f * diameter, diameter),
                                          Color(0, 0, 0), LAYER_GUI + 1);
```

Comment:
Don't forget to change the layer to something lower than LAYER_GUI or else the effect goes above the pause menu gui. I recommend LAYER_GUI-5

Reply:
Thanks for the hint!